### PR TITLE
docs: add flow charts for PXE and installation

### PR DESCRIPTION
**Pull request by andrewrynhard**
Adds flow charts describing the PXE boot and installation processes.

Signed-off-by: Andrew Rynhard <andrew@rynhard.io>

### CODE REVIEW

File: docs/website/content/docs/v0.1/Guides/flow.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    %% Chart
+
+    Start-->IsOn;
+    IsOn--Yes-->End;
+    IsOn--No-->PrepPowerOn;
+
+    PrepPowerOn--PXE Boot--->IsAccepted;
+
+    IsAccepted--Yes--->IsRegistered;
+    IsAccepted--No--->PrepPowerOff-->End;
```
Comment (smira) [inline docs/website/content/docs/v0.1/Guides/flow.md:41]: I believe we don't poweroff !accepted servers

**Review by smira — APPROVED** [review]
:heart:

**Comment by andrewrynhard** [comment]
> ❤️

Is this chart accurate? I don't mind iterating on it more to get it right.

**Comment by smira** [comment]
> > heart
> 
> Is this chart accurate? I don't mind iterating on it more to get it right.

lgtm, it doesn't cover PXE boot once, but it might be overkill to show here

**Comment by andrewrynhard** [comment]
> > > heart
> > 
> > 
> > Is this chart accurate? I don't mind iterating on it more to get it right.
> 
> lgtm, it doesn't cover PXE boot once, but it might be overkill to show here

Yeah I figured we could get the primary use case documented for now.

**Pull request merged**